be/src/vec/data_types/serde/data_type_ipv4_serde.cpp

```diff
@@ -0,0 +1,81 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "data_type_ipv4_serde.h"
+
+#include <arrow/builder.h>
+
+#include <type_traits>
+
+#include "gutil/casts.h"
+#include "vec/columns/column_const.h"
+
+namespace doris {
+namespace vectorized {
+
+void DataTypeIPv4SerDe::write_column_to_arrow(const IColumn& column, const NullMap* null_map,
```

Comment:
here no need to impl this function, becase it has same behavor with DataTypeNumberSerde<UInt32>